Stripe/STPAPIClient.m

```diff
@@ -449,3 +463,55 @@ - (void)stopPollingSourceWithId:(NSString *)identifier {
 }
 
 @end
+
+#pragma mark - Customers
+
+@implementation STPAPIClient (Customers)
+
++ (STPAPIClient *)apiClientWithEphemeralKey:(STPEphemeralKey *)key {
+    STPAPIClient *client = [[self alloc] init];
+    client.apiKey = key.secret;
```

Comment:
Hmm, should we make them not class methods? None of the other STPAPIClient methods are. This seems potentially limiting if we want to add config options for customer operations in the future.